qa/setup_packages.py

```diff
@@ -313,12 +313,12 @@ def get_pyvers_name(self, url, cuda_version):
                     return ret
         return ""
 
-all_packages = [PlainPackage("opencv-python", ["4.2.0.32"]),
+all_packages = [PlainPackage("opencv-python", ["4.4.0.42"]),
                 CudaPackage("cupy",
-                        { "100" : ["7.5.0"] },
+                        { "100" : ["7.8.0"] },
                         "cupy-cuda{cuda_v}"),
                 CudaPackage("mxnet",
-                        { "100" : ["1.5.1"] },
+                        { "100" : ["1.7.0"] },
```

Comment:
On the mxnet page I see 1.6 as the latest.

Reply:
https://pypi.org/project/mxnet-cu100/ - it shows 1.7